_posts/2019-06-21-my-infatuation-with-elm.markdown

```diff
@@ -0,0 +1,76 @@
+# My Infatuation With Elm
+
+by Cameron Gera (@gera-cameron)
+
+[Elm](https://elm-lang.org/) is a function language that is super reliable and offers NO runtime exceptions! It compiles down to vanilla javascript so it makes it easy to work in all browsers.
+There are a lot of incredible things in Elm, but I am going to touch on the following:
+
+- Tooling
+- Compiler
+- Easy to start with an existing application
+
+But before I get there, I'd like to tell you my Elm origin story.
+I have been working with Elm for a year and a half, but I feel like I've known it my entire life.
+I'm not going to lie, in the beginning it was tough to understand the architecture and the syntax since I was coming from Angular and Javascript as the main front-end languages I was familiar with.
+But over time and many compiler errors, things started to click in ways I never really thought would be possible.
+As a full-stack engineer, Elm makes me want to shift and become a front-end engineer full time.
+Enough about me, let's get started!
+
+## Tooling
+This is a section that I could just write a whole blog about.
+The tooling is one of my favorite things about Elm.
+I'll start with the formatter, [Elm Format]().
+This formatter is uniquely opinionated and not very customizable.
+What I mean by **uniquely** opinionated is that it will allow you to have a line that extends past your desired buffer length (which is frustrating at times), but it formats very spaciously.
```

Comment:
```suggestion
One of the frustrating parts, when I say **uniquely** opinionated, is that it will allow you to have a line that extends past your desired buffer length.
```